src/udev/cdrom_id/cdrom_id.c

```diff
@@ -988,6 +1020,31 @@ static int cd_media_toc(int fd) {
         return 0;
 }
 
+static int open_drive(Context *c) {
+        _cleanup_close_ int fd = -1;
```

Comment:
maybe assert(c->fd == -1) to ensure we don't leak FDs?

Reply:
Done.